From bdaf729356b22fa3302f982dec5aab3607411c80 Mon Sep 17 00:00:00 2001
From: Sandor Yu <Sandor.yu@nxp.com>
Date: Fri, 13 Jul 2018 15:14:06 +0800
Subject: [PATCH 4161/5242] MLK-18726-2: hdmi audio: HDMI 1.4 CTS 7-31 pass

commit  d08bfa1f79aa7214de31cddb0aba2b53b707ac5a from
https://source.codeaurora.org/external/imx/linux-imx.git

CT bits in Audio info frame shall always be set to 0.

Signed-off-by: Sandor Yu <Sandor.yu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/gpu/drm/imx/hdp/imx-hdp-audio.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/imx/hdp/imx-hdp-audio.c b/drivers/gpu/drm/imx/hdp/imx-hdp-audio.c
index e77bc90..dace3cc 100644
--- a/drivers/gpu/drm/imx/hdp/imx-hdp-audio.c
+++ b/drivers/gpu/drm/imx/hdp/imx-hdp-audio.c
@@ -70,7 +70,7 @@ static void imx_hdmi_audio_avi_set(state_struct *state,
 	hdmi_audio_infoframe_init(&frame);
 
 	frame.channels = channels;
-	frame.coding_type = HDMI_AUDIO_CODING_TYPE_PCM;
+	frame.coding_type = HDMI_AUDIO_CODING_TYPE_STREAM;
 
 	ret = hdmi_audio_infoframe_pack(&frame, buf + 1, sizeof(buf) - 1);
 	if (ret < 0) {
-- 
1.7.9.5

